modules/openxAnalyticsAdapter.js

```diff
@@ -681,28 +681,23 @@ function updateLoadedAdSlotsInfo(auctionId, adUnitCode, adPosition) {
 }
 
 function getLoadedAdUnitCodes(auctionId) {
-
   return (!auctionId || !loadedAdSlots[auctionId] || typeof loadedAdSlots[auctionId] !== 'object')
     ? [] : Object.keys(loadedAdSlots[auctionId]);
 }
 
 function pushAdPositionData(auctionId) {
-
-  if(auctionId && eventStack[auctionId] && eventStack[auctionId].events) {
-
+  if (auctionId && eventStack[auctionId] && eventStack[auctionId].events) {
```

Comment:
Do we know if Prebid uses the latest babel? 

This could be simplified to `if (auctionId && eventStack?.[auctionId]?.events)`. If not then this is fine.